### PR TITLE
Added basic profile for folgertech

### PR DESCRIPTION
**Pull request by nallath**
As per info provided by Paul Bussiere

### CODE REVIEW

File: resources/definitions/folgertech_FT-5.def.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "id": "FolgerTech_FT5",
```
Comment (Ghostkeeper) [inline resources/definitions/folgertech_FT-5.def.json]: Not that we use this ID any more, but it is not consistent with the file name.

Comment (nallath) [inline resources/definitions/folgertech_FT-5.def.json]: Forgot to remove that from the file.

**Comment by Ghostkeeper** [comment]
Thanks for your contribution, Paul (and Nallath)!

**Comment by fieldOfView** [comment]
Does that mesh really need to be 2 Mb? It seems like it must have lots of unnecessarily detailed geometry, because the actual model looks fairly simple.

**Comment by nallath** [comment]
Probably not, but I just added the file as I got it. My modeling skills are quite bad, so it would have taken me quite a while to fix it.

**Pull request merged**